frontend/lib/src/useBasicWidgetState.ts

```diff
@@ -61,18 +58,16 @@ export interface UseValueWSourceArgs<
 }
 
 /**
- * A React hook that makes the simplest kinds of widgets very easy to
- * implement.
+ * A React hook that makes the simplest kinds of widgets very easy to implement.
+ * Use the clientState version when the widget does not have a .setValue on its
```

Comment:
I love the comment ❤️ Maybe one additional half sentence to add `"...otherwise use the useBasicWidgetState` hook.